submissions/chipoHameja/js/multiples.html

```diff
@@ -0,0 +1,12 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="multiples.js"></script>

```

Comment:
script link should be at the end of your html tag, thats another good coding practise...